src/covid_shared/paths.py

```diff
@@ -33,19 +34,14 @@
 WEBSCRAPER_ROOT = Path('/ihme/covid-19/webscrape')
 
 MASK_USE_OUTPUT_ROOT = Path('/ihme/covid-19/mask-use-outputs')
```

Comment:
Are we sure these deletions aren't going to impact any other users? I'd leave these folders for now.